x-pack/legacy/plugins/watcher/common/models/action/action.test.js

```diff
@@ -35,43 +31,13 @@ describe('action', () => {
         };
       });
 
-      it(`throws an error if no 'id' property in json`, () => {
```

Comment:
I am not the last one that changed the code so I am guessing that if your removed all these tests is because we don't need those validations anymore in the UI.

Reply:
The original implementation threw `boom` errors when these properties were missing, which I believe means it was making assertions against the response from Elasticsearch when retrieving a watch with actions. I removed this logic because it both seems like an extreme corner case and the user also wouldn't have a clear way of fixing the problem.

Reply:
I see. Sounds more like an api integration tests indeed. 👍

Reply:
💯 I was thinking the same thing!